components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -334,15 +334,14 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
         if (formValues.type === 'user' || formValues.type === 'team') {
           this.addIdentityControl();
         } else if (formValues.type === 'token') {
-          this.addNameControl();
+          this.addNameControl(formValues.identityProvider);
```

Comment:
In order to know which validation type should be added, we pass in what the identity provider is